connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorRestartApiIntegrationTest.java

```diff
@@ -63,59 +66,71 @@
     private static final int ONE_WORKER = 1;
     private static final int NUM_TASKS = 4;
     private static final int MESSAGES_PER_POLL = 10;
-    private static final String CONNECTOR_NAME = "simple-source";
+    private static final String CONNECTOR_NAME_PREFIX = "simple-source";
```

Comment:
Nit: would a delimiter here help?
```suggestion
    private static final String CONNECTOR_NAME_PREFIX = "simple-source-";
```

Reply:
I agree and I fixed it as part of recent tests changes,
`    private static final String CONNECTOR_NAME_PREFIX = "conn-";
`